Caches/test/EteTests/testRunner.py

```diff
@@ -62,81 +59,15 @@ def chechTestResult(goldenPath, testResult):
     print("Executable main file: % s" % args.executable)
     print("Directory containing tests: % s" % args.test_dir)
 
-    testResult = runTest(args.executable, args.test_dir + "test1.dat")
-    checkResult = chechTestResult(args.test_dir + "golden/test1.ans.dat", testResult)
-    print(checkResult)
+    test_files = [f for f in os.listdir(args.test_dir) if f.endswith('.dat') and not f.startswith('golden')]
```

Comment:
All variables are name using CamelCase.
Could you please raname test_files variable (or raname all other variables in one non-CamelCase style)?